docker/push-all.sh

```diff
@@ -24,7 +24,7 @@ CLEANUP="false"
 
 # list of images we are tagging & pushing
 IMAGES=()
-IMAGES_VPP=("cni" "ksr" "stn" "crd" "vswitch")
+IMAGES_VPP=("cni" "ksr" "stn" "crd" "vswitch" "dev-switch")
```

Comment:
`dev-switch` should be `dev-vswitch`.. but I think we don't wont to push the dev image anyway, it is too large.

Reply:
I just preserved existing functionality, e.g there is option for dev upload, so I just did

```
	if [ "${DEV_UPLOAD}" != "true" ] && [ "$IMAGE" == dev* ]; then
		continue
	fi
``` instead of having the dev-vswitch in separate upload at bottom

Reply:
ah, okay :) haven't noticed that